modules/ROOT/pages/6.0-release-notes-parsing-and-serialisation.adoc

```diff
@@ -0,0 +1,24 @@
+= TinyMCE 6.0 parsing and serialisation changes
+:navtitle: TinyMCE 6.0 parsing and serialisation changes
+:description: TinyMCE 6.0 parsing and serialisation changes
+:keywords: releasenotes, parsing, serialisation
+
+This section documents the single change made to {productname} 6.0 parsing and serialisation.
+
+* xref:parsing-and-serialisation[Parsing and serialisation]
+* xref:changed-parsing-and-serialisation[Changed parsing and serialisation]
+
+// tag::parsing-and-serialisation[]
+[[parsing-and-serialisation]]
+== Parsing and serialisation
+
+
+[[changed-parsing-and-serialisation]]
+=== Changed parsing and serialisation
+
+[[rgb-colors-no-longer-converted-to-hex-values]]
+==== RGB colors no longer converted to hex values
+
+With the removal of the, now deprecated, `toHex` APIs, RGB colors are no longer converted to hex values when parsing or serializing content.
```

Comment:
Do we support RGB colors now?

Reply:
The UI colour support hasn't changed. This just impacts what happens when content is loaded or pasted.

Reply:
The conversion was added (if I remember correctly) to support different colors in the pallette that textbox.io supported. We still have the hex conversion code in connections so that the same config can be used for textbox.io and TinyMCE.

Reply:
I can't comment on why `toHex` was deprecated, but this conversion was one of the main things it did. If you're talking about the colour picker palette that shouldn't have changed in 6?

Reply:
Ok, probably unrelated to what I was thinking of then.